### ISSUE TITLE
AS -Incorrect balance for Qtum Electrum

### ISSUE THREAD

**Issue by gingervik** [BeamMW/beam-ui#417]
If user is connected to QTUM Electrum node Beam Wallet displays incorrect balance
![image](https://user-images.githubusercontent.com/28015224/101319137-81339280-3872-11eb-851b-eea751a9ee50.png)
![image](https://user-images.githubusercontent.com/28015224/101319147-8690dd00-3872-11eb-8af1-ce631dab7eaf.png)

**Comment by gingervik** [comment]
@Sergei-Beam please, recheck

**Comment by Sergei-Beam** [comment]
Reproduced
After restoring Qtum via seed wallet shows 0 Balance,
But if user request coins for address form Beam wallet than delivered coins shown in Qtum Balance. Example, I requested testnet coins from the faucet.

Also, it looks strange because Qtum wallet also didn't show coins which were got to address to Qtumadress from Beam wallet on the same seed.

![image](https://user-images.githubusercontent.com/44496108/105218576-612c0a00-5b66-11eb-9790-9b421c47bea3.png)
